docs/developers/bitesize/workflowExecution/majorActors.md

```diff
@@ -0,0 +1,53 @@
+# Workflow Execution: Major Actors
+
+* **Bitesize Documentation:** Covering the main actors involved in workflow execution.
+* **Word Count:** 195
+
+## High Level Overview
+
+![high level overview diagram](WorkflowExecutionHighLevelOverview.png)
+
+## Actors and their Purposes
+
+### WorkflowManagerActor
+
+The `WorkflowManagerActor` is responsible for:
+
+* Polling the `WorkflowStore` at pre-configured intervals.
+* Starting new workflows
+* Tracking, supervising and aborting running workflows
+* Parent actor for all `WorkflowActor`s
+
+### WorkflowActor(s)
+
+Each `WorkflowActor` is responsible for:
+
+* Running a single workflow.
+* Starting jobs and sub-workflows as soon as they are able to run.
+    * Based on values in the (in-memory) ValueStore and ExecutionStore objects.
+* Parent actor for all `EngineJobExecutionActor`s and `SubWorkflowExecutionActor`s.
```

Comment:
With at least 55 words to spare (not sure how strictly the word limit is enforced) is it worth mentioning the MWDA, WIA, and/or WEA?